### PR TITLE
Update install

### PR DESCRIPTION
**Pull request by cyberqueen-meg**
Add support for arcolinux and blackarch

# Pull Request

Thanks for submitting a PR! Please fill in this template where appropriate:

## Category
*Core Functionality*

## Feature/Issue Description
**Adding install support for BlackArch and ArcoLinux Linux distributions**

**Q:** Give a technical rundown of what you have changed (if applicable)
**Adding support to the installer for BlackArch and ArcoLinux, both Linux distributions based directly off of Arch Linux.**

## Test Cases
**I tested the install on BlackArch and ArcoLinux**

## Wiki Page
*N/A*

### CODE REVIEW

**Comment by wheatley** [comment]
hey @cyberqueen-meg thanks for the PR, 

While I'm downloading a copy of the ISOs for BlackArch and ArcoLinux did you want to check out some of the comments that have been made.

also feel free to join our discord and start chatting with the community. https://discord.gg/r3HnzQzy

File: install [hunk 1]
```diff
@@ -87,7 +87,7 @@ install_linux() {
 		DISTRO_ID=$(cat /etc/os-release | grep ID= | grep -v "BUILD" | grep -v "IMAGE" | cut -d= -f2-)
 		if [ "${DISTRO_ID}" = 'kali' ]; then
 			Distro='Kali'
-		elif [ "${DISTRO_ID}" = 'arch' ] || [ "${DISTRO_ID}" = 'artix' ] || [ "${DISTRO_ID}" = 'manjaro' ]; then
+		elif [ "${DISTRO_ID}" = 'arch' ] || [ "${DISTRO_ID}" = 'artix' ] || [ "${DISTRO_ID}" = 'manjaro' ] || [ "${DISTRO_ID}" = 'blackarch' ] || [ "${DISTRO_ID}" = 'blackarch' ]; then
```
Comment (bcoles) [inline install:90]: Why is `[ "${DISTRO_ID}" = 'blackarch' ]` listed twice?

**Review by bcoles — COMMENTED** [review]
> Adding install support for BlackArch and ArcoLinux Linux distributions

This PR adds BlackArch twice and ArcoLinux is nowhere to be found...

**Comment by cyberqueen-meg** [comment]
OMG, that's an error on my part, let me fix that

**Comment by cyberqueen-meg** [comment]
Closing this PR because I can't figure out how to add the new edit to this PR - see my new req

**Comment by bcoles** [comment]
#2487

**Pull request closed without merging**